style.css

```diff
@@ -106,8 +110,118 @@
     .btn{
         text-align: left;
         margin-top: 38px;
-        /*margin-right: 0;*/
-        /*margin-left: 20px;*/
+        margin-left: 20px;
         border: none;
     }
+}
+.recent_posts{
+    margin-top: 59px;
+    background-color: #EDF7FA;
+    margin-bottom: 38px;
+}
+.recent_heading{
+    display: flex;
+}
+.view{
+    display: none;
+}
+.post{
+    text-align: center;
+    font-family: 'Heebo', sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    color: #21243D;
+    margin-left: 135px;
+    margin-right: 135px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+}
+.post_one{
```

Comment:
Margin, Font (size, color), Vertical line, etc issues are there in this block. Please match with design and fix them.

![Screenshot 2023-03-13 at 2 49 15 PM](https://user-images.githubusercontent.com/38748298/224658671-68606dde-81cd-44cc-9aaf-d248fae63672.png)